### DISCUSSION TITLE
폰트 적용에 궁금한 점이 있습니다.

### DISCUSSION THREAD

**Discussion by dmchoi77** [orioncactus/pretendard#183] [Q&A] [answered]
Discussions를 읽어보다가 궁금한 점이 있어 질문드립니다.
```
<head>
...
     <link rel="preconnect" href="https://cdn.jsdelivr.net" crossorigin="anonymous">
     <link rel="preload" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/variable/pretendardvariable-dynamic-subset.css" />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/variable/pretendardvariable-dynamic-subset.css" />
...
</head>
```

이렇게 적용을 하라고 말씀하셨는데, 같은 리소스에 대해서 preload와 stylesheet를 함께 사용하는 이유가 무엇일까요?
또한 위와 같이 사용할 경우 pretendardvariable-dynamic-subset.css을 두 번 요청하게 되더라구요..
그래서 stylesheet만 사용하면 되는게 아닌지 궁금증이 생겨 질문드립니다!

**Comment by orioncactus** [comment, marked as answer]
preload는 웹사이트에서 파일을 우선적으로 다운로드하도록 하는 속성으로, 페이지를 로드할 때 Pretendard를 우선적으로 불러오게 함으로써 보통 폰트를 늦게 로드할 때 기본 폰트를 렌더링하며 나타날 수 있는 깜빡임 현상을 줄이기 위해 씁니다.

이어서 preload 앞에 있는 preconnect는 도메인에 사전 연결을 할 수 있도록 하는 속성으로, 일반적으로 preload를 정의할 때 같이 쓰는 편입니다.

preload와 preconnect에 대한 자세한 설명은 이 아티클에서 자세히 다루고 있으니, 참고해보시면 좋을 듯합니다 ☺️: https://blog.banksalad.com/tech/font-preload-on-safari/
